### PR TITLE
Add role aliases to allow our mispelled role :(

### PR DESCRIPTION
**Pull request by shanear**
connects #1275

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -2,19 +2,30 @@ class User < ActiveRecord::Base
...
+  FUNCTION_ALIASES = {
+    "Manage Claim Establishme" => ["Manage Claim Establishment"]
+  }.freeze
+
   def username
     css_id
   end
 
+  def roles
+    (@roles || []).inject([]) do |result, role|
```
Comment (joofsh) [inline app/models/user.rb]: haha wow, I've never seen an array <=> array inject before. Pretty cool ruby-fu 👍

File: app/models/user.rb [hunk 2]
```diff
@@ -2,19 +2,30 @@ class User < ActiveRecord::Base
...
 
   TASK_TYPE_TO_ROLES = {
     EstablishClaim: { employee: "Establish Claim", manager: "Manage Claim Establishment" }
   }.freeze
 
   FUNCTIONS = ["Establish Claim", "Manage Claim Establishment", "Certify Appeal", "CertificationV2"].freeze
 
+  # Because of the funciton character limit, we need to also alias some functions
+  FUNCTION_ALIASES = {
+    "Manage Claim Establishme" => ["Manage Claim Establishment"]
```
Comment (joofsh) [inline app/models/user.rb]: 😢

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -21,6 +21,22 @@
...
 
+  context "#roles" do
+    subject { user.roles }
+
+    context "when roles is nil" do
+      before { user.roles = nil }
+      it { is_expected.to eq([]) }
+    end
+
+    context "when has a role alias" do
```
Comment (joofsh) [inline spec/models/user_spec.rb]: Can we add another test checking that a regular role (or 2) do properly get returned (not coming from an alias)?

File: app/models/user.rb (hunk 2 above)
Comment (kierachell) [inline app/models/user.rb]: claims not claim

**Pull request merged**